backend/compact-connect/lambdas/common-python/cc_common/data_model/client.py

```diff
@@ -255,6 +260,8 @@ def create_provider_privileges(
         :param jurisdiction_postal_abbreviations: The list of jurisdiction postal codes
         :param license_expiration_date: The license expiration date
         :param compact_transaction_id: The compact transaction id
+        :param existing_privileges: The list of existing privileges for this user. Used to track the original issuance
```

Comment:
Security: Offline with @jusdino.

Reply:
@jusdino One other thing: I think the create_staff_user.py script is broken. I made some updates on this PR, along with some README updates which I welcome your thoughts on!

https://github.com/csg-org/CompactConnect/pull/355